render/render.js

```diff
@@ -497,18 +499,8 @@ module.exports = function($window) {
 		return false
 	}
 
-	function copy(data) {
-		if (data instanceof Array) {
-			var output = []
-			for (var i = 0; i < data.length; i++) output[i] = data[i]
-			return output
-		}
-		else if (typeof data === "object") {
-			var output = {}
-			for (var i in data) output[i] = data[i]
-			return output
-		}
-		return data
+	function assign(target, source) {
```

Comment:
`for in` loops are considerably slower for arrays & objects, I'm not sure about this change.

Arrays: https://jsbin.com/jorodun/3/edit?js,console
Objects: https://jsbin.com/hucimo/edit?js,console,output

Seems like it should retain the sniffing, but just use `.forEach` for iteration & copying properties/elements.